policyengine_canada/variables/gov/provinces/qc/qc_living_in_northern_villages.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_canada.model_api import *
+
+
+class qc_living_in_northern_villages(Variable):
+    value_type = bool
+    entity = Person
```

Comment:
Geographic variables are household-level
```suggestion
    entity = Household
```